src/signMessage.c

```diff
@@ -104,66 +95,6 @@ UX_FLOW(ux_4_fields,
   &ux_reject_step
 );
 
-static int process_message_body(uint8_t* message_body, int message_body_length, MessageHeader* header, field_t* fields) {
```

Comment:
Removing this `static` exposes a bug. Afterward, clicking `Approve` causes the device to hang.